extra_model/_cli.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from pathlib import Path
+from typing import Dict
+
+import click
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command()
```

Comment:
this is great 👍 improved and simplified compared to whats in the original version